GUI/MainWindow.xaml.cs

```diff
@@ -1489,6 +1489,11 @@ private void MenuItem_Click_6(object sender, RoutedEventArgs e)
             System.Diagnostics.Process.Start(GlobalVariables.DataDir);
         }
 
+        private void MenuItem_Proxl(object sender, RoutedEventArgs e)
```

Comment:
move this up one method spot and put it right below the slack click
![image](https://user-images.githubusercontent.com/16841846/69552730-963dcc00-0f64-11ea-8be3-a7c69b3f952b.png)